tests/drivers/i2s/i2s_api/src/common.c

```diff
@@ -111,7 +112,7 @@ static int tx_block_write_slab(const struct device *dev_i2s, int att, int err,
 	char tx_block[BLOCK_SIZE];
 	int ret;
 
-	fill_buf((uint16_t *)tx_block, att);
+	fill_buf((int16_t *)tx_block, att);
```

Comment:
Why is change made?

Reply:
the fill_buf has changed by kuma to below
```
static void fill_buf(int16_t *tx_block, int att)
```

Reply:
There are a bunch of other places where this function is called with `uint16_t`. Do we need to change all instances?